assets/javascripts/card-form-customization.js

```diff
@@ -58,6 +58,24 @@ function getDesignFormValues() {
   return formValues;
 }
 
+function handleFontChange() {
+  const fontName = document.getElementById('custom_sf_font_name');
+
+  if (fontName.value == 'Others') {
+    document.getElementById('custom_sf_custom_font_name').style.display = null
+  }
+
+  fontName.addEventListener('change', (event) => {
+    if (event.target.value == 'Others') {
+      document.getElementById('custom_sf_custom_font_name').style.display = null
+    } else {
+      const customFontName = document.getElementById('custom_sf_custom_font_name');
+      customFontName.value = "";
+      customFontName.style.display = "none";
```

Comment:
In this function, we're using both single and double quotes.

Reply:
Changed it to single quotes